build.gradle

```diff
@@ -278,6 +320,10 @@ dependencies {
         force = true
     }
 
+    contractTestCompile(group: 'org.yaml', name: 'snakeyaml', version: '1.23') {
```

Comment:
I am wondering if adding `snakeyaml` to `buildscript` scope would resolve issue. It helps in document-api
```
buildscript {
    repositories {
        jcenter()
    }
    dependencies {
        classpath("org.yaml:snakeyaml:1.23")  // <- this is new line needed to override gradle snakeyaml lib
        classpath("net.serenity-bdd:serenity-gradle-plugin:2.0.11")
    }
}
```

Reply:
Cool, will try it out.

Reply:
Can't reproduce the problem to test if it works.  I suggest we leave it as it is until someone gets the problem again and see if this fixes it.